libraries/redcore/toolbar/builder.php

```diff
@@ -47,21 +47,24 @@ public static function createCopyButton($task, $class = '')
 	/**
 	 * Create a new button.
 	 *
-	 * @param   string  $task   The task name.
-	 * @param   string  $class  A css class to add to the button.
+	 * @param   string  $task      The task name.
+	 * @param   string  $class     A css class to add to the button.
+	 * @param   bool    $isOption  Is button a menu option?
 	 *
 	 * @return  RToolbarButtonStandard  The button.
 	 */
-	public static function createNewButton($task, $class = '')
+	public static function createNewButton($task, $class = '', $isOption = false)
 	{
-		if (empty($class))
-		{
-			$class = 'btn-success';
-		}
-
-		else
+		if (!$isOption)
```

Comment:
Yup, you are right. I intend to make it that way, not sure why I had brain fart there